packages/web/src/javascripts/Controllers/PaneController/PaneController.ts

```diff
@@ -136,6 +140,30 @@ export class PaneController extends AbstractViewController implements InternalEv
       this.setCurrentNavPanelWidth(this.preferences.getValue(PrefKey.TagsPanelWidth, MinimumNavPanelWidth))
       this.setCurrentItemsPanelWidth(this.preferences.getValue(PrefKey.NotesPanelWidth, MinimumNotesPanelWidth))
     }
+    if (event.type === ApplicationEvent.LocalPreferencesChanged) {
+      this.listPaneExplicitelyCollapsed = this.preferences.getLocalValue(
+        LocalPrefKey.ListPaneCollapsed,
+        LocalPrefDefaults[LocalPrefKey.ListPaneCollapsed],
+      )
+      this.navigationPaneExplicitelyCollapsed = this.preferences.getLocalValue(
+        LocalPrefKey.NavigationPaneCollapsed,
+        LocalPrefDefaults[LocalPrefKey.NavigationPaneCollapsed],
+      )
+      const screen = this._isTabletOrMobileScreen.execute().getValue()
```

Comment:
this will now run after every `setLocalValue` call, no? i think it makes more sense to have a boolean in the class to make sure this only runs once since its supposed to be initialization logic.